### PR TITLE
gradient accumulation for training & finetuning

### PR DESCRIPTION
**Pull request by raymondzmc**
Added argument for gradient accumulation in training & fine-tuning.

### CODE REVIEW

**Comment by raymondzmc** [comment]
Deleted, forgot to remove trace points
